File: palindromo.js
let texto = prompt("Digite uma palavra ou frase:");

let tratado = texto.replace(/ /g, "").toLowerCase();
let invertido = tratado.split("").reverse().join("");

alert(tratado === invertido
  ? `"${texto}" é um palíndromo.`
  : `"${texto}" não é um palíndromo.`
);

File: palindromo.py
texto = input("Digite uma palavra ou frase: ")

# remove espaços e transforma em minúsculas
tratado = texto.replace(" ", "").lower()

invertido = tratado[::-1]

if tratado == invertido:
    print(f'"{texto}" é um palíndromo.')
else:
    print(f'"{texto}" não é um palíndromo.')
